tools/nightly_build_and_test_stats.groovy

```diff
@@ -728,8 +885,13 @@ node('worker') {
                     }
                 }
 
+                def reproducibilityText = "Reproducibility: N/A"
```

Comment:
If it's for a version that is not in reproducibleBuilds, then lets not output the "Reproducibility: N/A" text, as jdk8,11,17 will never be there...

Reply:
Okie dokie.

Reply:
Done, and I adjusted the spaces too so the kerning is ok.